test-sender/src/main.rs

```diff
@@ -81,13 +81,15 @@ async fn start_sender(config: TestSenderConfig) {
             let content_topic = format!("/{}/0/{}/proto", config.radio_name, topic);
             let content_topic = WakuContentTopic::from_str(&content_topic).unwrap();
 
+            let nonce = config.nonce.clone().unwrap().parse::<i64>().unwrap();
```

Comment:
I think we shouldn't use need a configured nonce, we can just use timestamp created on L75

Reply:
maybe then replace the timestamp used for building graphcast message to the nonce here, since you are overwriting it anyway later on